Makefile

```diff
@@ -29,7 +29,7 @@ install-base-pip-packages:
 
 .PHONY: install-detectron2
 install-detectron2:
-	pip install "detectron2@git+https://github.com/facebookresearch/detectron2.git@v0.6#egg=detectron2"
+	pip install "detectron2@git+https://github.com/facebookresearch/detectron2.git@78d5b4f335005091fe0364ce4775d711ec93566e"
```

Comment:
We have a problem:
The detectron v.06 requires black 21.4b2 which have a problem with click
https://stackoverflow.com/questions/71673404/importerror-cannot-import-name-unicodefun-from-click
So, updating black to 22.3.0  (and fix the click problem) or greater breaks the compatibility with detectron2.
I forgot this, so, this is why we need to change the detectron2 version

Reply:
We had similar conversation in https://github.com/Unstructured-IO/pipeline-document-layout/pull/8

Reply:
Okay that makes sense. That commit is from Dec 2022 and the 0.6 (the latest tag) is from Dec 2021, so they seem to have stopped cutting tags. We may want to update that in `unstructured` as will, since we're pinned to `0.6` there:

- https://github.com/Unstructured-IO/unstructured/blob/main/Makefile#L58